src/components/Button.js

```diff
@@ -1,12 +1,24 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+let defaultBgColor = {
+  backgroundColor: 'orange'
+};
+
+
 class Button extends Component {
   render() {
+    const wideStyle = (this.props.wide) ? 'wideStyle' : '';
+    let bgColor;
+    if (!this.props.color) {
+      bgColor = defaultBgColor;
+    } else {
+      bgColor = { backgroundColor: this.props.color };
+    }
```

Comment:
Let's set defaultProps for the color prop and then we don't need this logic.
Also set color's propType.